cf-atcp-contact/deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Script to prepare and deploy function.
+
+set -e
+
+if (( $# != 1 )); then
+  echo 'usage: deploy.sh <GCP_PROJECT>'
+  exit 1
+fi
+
+PROJECT_ID="$1"
+SECRET_ID='cf-atcp-contact'
+CF_NAME='cf-atcp-contact'
+SA_ACCT="cf-atcp-contact@$PROJECT_ID.iam.gserviceaccount.com"
+
+echo 'Rendering env vars file...'
+gcloud --project=$PROJECT_ID \
+  secrets versions access latest \
+  --secret=$SECRET_ID > env.yaml
+
+gcloud --project=$PROJECT_ID functions deploy $CF_NAME \
+  --entry-point org.broadinstitute.ddp.cf.ATCPStudyContact \
+  --trigger-http \
+  --allow-unauthenticated \
+  --service-account $SA_ACCT \
+  --env-vars-file env.yaml \
+  --region us-central1 \
+  --runtime java11 \
+  --memory 256MB \
+  --timeout 1m \
+  --max-instances 3
```

Comment:
Since this is a publicly-facing HTTP endpoint, we should put limits on stuff. Looking at charts, I think these numbers should be good enough.

![Screen Shot 2021-02-09 at 9 10 01 PM](https://user-images.githubusercontent.com/5713833/107454611-59cf9f00-6b1b-11eb-89fe-bfda156803c9.png)

![Screen Shot 2021-02-09 at 9 10 16 PM](https://user-images.githubusercontent.com/5713833/107454629-618f4380-6b1b-11eb-8354-c9e464927f0a.png)

Reply:
Sounds reasonable.